js/constant/our-project-constant.js

```diff
@@ -0,0 +1,34 @@
+var project=[
+    {
+        name: 'Circle',
+        description : 'Circle is an online platform for creating and sharing meaningful information and knowledge within the like minded community. The platform is for connecting learners and experts together and creating a positive influence in the community. ',
+        url : 'https://circleapp.insightjet.com',
+        backgroundImage : {"background-image":"linear-gradient(to bottom, rgba(0, 0, 0, 0), rgb(0, 0, 0)),url('images/our-projects/Circle.png')"}
```

Comment:
you need to send me these images.. it will not read ur images folder. I should upload them to cloudinary first.

Reply:
done